src/zypper-plugins/enabled_repos_upload.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from os import environ
+import sys
+import logging, traceback
+
+from katello.repos import upload_enabled_repos_report
+from katello.enabled_report import EnabledReport
+from subscription_manager.repolib import ZypperRepoFile
+
+from zypp_plugin import Plugin
+
+class KatelloEnabledUploadZyppPlugin(Plugin):
+    def __init__(self):
+        Plugin.__init__(self)
+
+    def PLUGINEND(self, headers, body):
+        logging.info("Uploading Enabled Repositories Report")
+        try:
+            report = EnabledReport(ZypperRepoFile())
+            logging.info("Uploading Enabled Repositories Report11  ->  %s" % str(report))
+            upload_enabled_repos_report(report)
+        except:
+            logging.error("Unable to upload Enabled Repositories Report11 - %s" % traceback.format_exc())
+        self.ack()
+
+if "DISABLE_KATELLO_ZYPP_PLUGIN" in environ:
```

Comment:
this is the same environment variable used in package_upload - just wanted to make sure that you wanted to be able to disable both plugins with a single variable. I think it's OK since they are a pair

Reply:
I saw them as a pair. I am ok cutting it out also. I mean they can just uninstall the rpm right if they dont want it to run. You ok with remove ?

Reply:
Wasn't suggesting removal - I think it's fine. Just wanted to make sure that's the behavior we want and I'm good with the way it is if you are!